Firestore/core/src/local/lru_garbage_collector.cc

```diff
@@ -26,16 +26,18 @@
 #include "Firestore/core/src/local/target_data.h"
 #include "Firestore/core/src/model/document_key.h"
 #include "Firestore/core/src/util/log.h"
+#include "Firestore/core/src/util/statusor.h"
 
 namespace firebase {
 namespace firestore {
 namespace local {
 namespace {
 
-using firebase::firestore::api::Settings;
-using firebase::firestore::model::DocumentKey;
-using firebase::firestore::model::ListenSequenceNumber;
-using firebase::firestore::model::TargetId;
+using api::Settings;
+using model::DocumentKey;
+using model::ListenSequenceNumber;
+using model::TargetId;
+using util::StatusOr;
```

Comment:
Nit: within the PR, some source files qualify `util::StatusOr` and some rely on a `using` declaration like this. Should we pick one and make this consistent? (can certainly be done in a follow-up)

Reply:
There were a few stray references in `leveldb_persistence.cc` that I've fixed. Otherwise the difference is headers vs source files. In headers we can't have the using statements that shorten this.

Reply:
Yeah, I only meant source files. Thanks for fixing!